gpii/node_modules/settingsHandlers/src/settingsHandlerUtilities.js

```diff
@@ -335,6 +335,9 @@ gpii.settingsHandlers.readFile = function (options) {
 gpii.settingsHandlers.writeFile = function (content, options) {
     if (!options || !options.filename) {
         fluid.fail("writeFile: expected an options block defining filename and encoding");
+    } else if (!fs.existsSync(options.filename)) {
```

Comment:
There are two cases of non-existence.  The case here is when the solution is not installed and there is no settings file to write.  This is the flip side of using the solution to [GPII-2006](https://issues.gpii.net/browse/GPII-2006), where the reading of the file fails for the same reason.  @kaspermarkus wrote [above](https://github.com/GPII/universal/pull/461#pullrequestreview-927781):
> I tried running the NVDA acceptance tests in windows - but on a machine without NVDA installed - and they exploded (since it couldn't find the config file)...

That happens because the settings file is within a directory structure that doesn't exist if the solution is not installed, something like "...\nvda\nvda.ini"  There is no parent "nvda" directory.  I'll update the unit test for writing to a non-existent file to reflect that.

The other sense of non-existence is where the solution is installed, but needs to be run once to create the settings file.  In that case, writing (creating) the settings file should be supported.

We need to handle both cases.  The system should be sensitive to the context -- whether the solution is installed -- and react appropriately.